### PR TITLE
added image upload modals page

### PR DESCRIPTION
**Pull request by MaureenWanjiku**

### CODE REVIEW

**Review by pwnipc — CHANGES_REQUESTED** [review]
Separate the modals into two components
- pictureUpload
and
- pictureDelete

File: src/app/deletephoto.js
Comment (pwnipc) [inline src/app/deletephoto.js:1]: Make this a button 
image
And add an onClick event to it that pop's up the delete confirmation dialogue box
image

**Review by pwnipc — CHANGES_REQUESTED** [review]
have a look at the requested changes

File: src/app/deletephoto.js
Comment (MaureenWanjiku) [reply src/app/deletephoto.js:1]: Okay, working on the requested changes.

**Review by pwnipc — APPROVED** [review]

**Pull request merged**